src/cli/cli.cpp

```diff
@@ -6182,6 +6184,69 @@ template <> otError Interpreter::Process<Cmd("radiofilter")>(Arg aArgs[])
 }
 #endif
 
+#if OPENTHREAD_CONFIG_RADIO_TIME_STATS_ENABLE
+inline unsigned long UsToSInt(uint64_t aUs) { return ToUlong(static_cast<uint32_t>(aUs / 1000000)); }
+inline unsigned long UsToSDec(uint64_t aUs) { return ToUlong(static_cast<uint32_t>(aUs % 1000000)); }
+
+template <> otError Interpreter::Process<Cmd("radio")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+    if (aArgs[0] == "stats")
+    {
+        if (aArgs[1].IsEmpty())
+        {
+            otRadioStats radioStats;
+            uint64_t     totalTimeUs;
+            IgnoreError(otRadioStatsGet(GetInstancePtr(), &radioStats));
+
+            totalTimeUs = radioStats.mSleepTime + radioStats.mTxTime + radioStats.mRxTime + radioStats.mDisabledTime;
+            if (totalTimeUs == 0)
+            {
+                OutputLine("Total Time is 0!");
+            }
+            else
+            {
+                uint32_t txPercentInt = static_cast<uint32_t>(radioStats.mTxTime * 100 / totalTimeUs);
+                uint32_t txPercentDec =
+                    static_cast<uint32_t>((radioStats.mTxTime * 100 % totalTimeUs) * 100 / totalTimeUs);
+                uint32_t rxPercentInt = static_cast<uint32_t>(radioStats.mRxTime * 100 / totalTimeUs);
+                uint32_t rxPercentDec =
+                    static_cast<uint32_t>((radioStats.mRxTime * 100 % totalTimeUs) * 100 / totalTimeUs);
+                uint32_t sleepPercentInt = static_cast<uint32_t>(radioStats.mSleepTime * 100 / totalTimeUs);
+                uint32_t sleepPercentDec =
+                    static_cast<uint32_t>((radioStats.mSleepTime * 100 % totalTimeUs) * 100 / totalTimeUs);
+                uint32_t disabledPercentInt = static_cast<uint32_t>(radioStats.mDisabledTime * 100 / totalTimeUs);
+                uint32_t disabledPercentDec =
+                    static_cast<uint32_t>((radioStats.mDisabledTime * 100 % totalTimeUs) * 100 / totalTimeUs);
+
+                OutputLine("Radio Statistics:");
+                OutputLine("Total Time: %lu.%03lus", ToUlong(static_cast<uint32_t>(totalTimeUs / 1000000)),
+                           ToUlong((totalTimeUs % 1000000) / 1000));
+                OutputLine("Tx Time: %lu.%06lus (%lu.%02lu%%)", UsToSInt(radioStats.mTxTime),
```

Comment:
We can consider writing a helper method that `OutputRadioStatTime()` to output a time value and its percentage (and does percentage calculation)?